atlas_placement_hints/app/placement_hints.py

```diff
@@ -305,41 +311,84 @@ def isocortex(
     required=True,
     help="path of the directory to write. It will be created if it doesn't exist.",
 )
+@click.option(
+    "--thalamus-meshes-dir",
+    required=True,
+    help="""Path of the directory to use for either saving or loading thalamus
+    meshes. It will be created if it doesn't exist.""",
+)
+@click.option(
+    "--load-cut-thalamus-meshes",
+    required=False,
+    help="""(Optional) Flag to load your custom thalamus meshes, and then use
+    them to calculate placement-hints.""",
+    default=False,
+    is_flag=True,
+)
 @log_args(L)
+# pylint: disable=too-many-arguments
 def thalamus(
-    verbose, annotation_path, hierarchy_path, metadata_path, direction_vectors_path, output_dir
+    verbose,
+    annotation_path,
+    hierarchy_path,
+    metadata_path,
+    direction_vectors_path,
+    output_dir,
+    thalamus_meshes_dir,
+    load_cut_thalamus_meshes,
 ):
     """Generate and save the placement hints of the mouse thalamus.
 
-    Placement hints are saved under the names sepecified in `app/metadata/thalamus_metadata.json`.
-    Default to:
+    First, call this without passing '--load-cut-thalamus-meshes' to
+    create your region meshes, but not your placement-hints. Then, hand-cut
+    your meshes according to the documentation in
+    'atlas_placement_hints/layered_atlas.py::ThalamusAtlas'. Finally, call
+    this again while passing '--load-cut-thalamus-meshes'.
+
+    Placement hints are saved under the names specified in
+    `app/metadata/thalamus_metadata.json`. These default to:
 
     \b
     - `[PH]y.nrrd`
-    - `[PH]Rt.nrrd`, `[PH]VPL.nrrd`
+    - `[PH]layer_1.nrrd` (This is the "top-most" layer, equivalent to "RT".
+      This previously used the filename `[PH]RT.nrrd`)
```

Comment:
is there a reason you're calling it "layer 1" instead of RT?  I find the later easier to reason about

Reply:
Yes, it's a new requirement imposed by the BBP Atlas Pipeline: for any rules that apply region-specific customization (such as placement hints for the thalamus), the output files need to be named in a very particular way. If you have questions, slack me or look at the "Customize a pipeline rule" section of the BBP Atlas Pipeline README.